### PR TITLE
Mining Pool Rebalance and Loot Reworks, Pt. 1

### PR DESCRIPTION
**Pull request by Sun-Soaked**
## About The Pull Request
**A number of tweaks, rebalances, additions, ports, and partial removals of assorted mining loot:**
- Immortality Talisman spent some time with the ancient masters to better cultivate it's connection to the null zone. The Talisman can now be held in and triggered in pockets, allows for use of hands and some items while inside the shadow realm, and can set a custom warcry with alt-click!
-To compensate slightly, the active time and cooldown have both been amplified by 1.5.
- The throwing hook now has the ability to do weak, quick attacks in melee range. These are easily defeated by armor, but can be brutal against unprepared targets.
- Adds the Freeze Cube, a strange magical boomerang that freezes targets before bouncing back to the user. Remember to catch on return, unless you want to end up like your opponent.
- The possessed sword acquired via. Necropolis chests now has a custom aesthetic, increased damage, and the ability to slowly bleed up build affects akin to the cleaving saw. Devour your enemies with a friend by your side.
- Most unexpectedly, the cursed katana has received a strange power from an unknown patron! Beware, for the being that now lurks inside hungers for more than just the blood of your enemies.
- The Broodmother Tongue and Legion Spine crusher trophy effects have been significantly buffed, and now enjoy a slightly less useless place within the crusher extended universe.
- Pandora's "Hope" Talisman now provides armor akin to the Bone Talisman, alongside the mood boost.
- The Dark Blessing has been removed from chest loot, may be re-added later in this pr depending on a possible rework.
- The Jacob's ladder has been removed from the loot pool. You know what you did.
- Nobody seems to have actually noticed, but the paradox bag was by far the most robust storage solution in the game, and could contain pretty much everything you'd ever need. It has had it's original storage cap reduced by half- still a metric ton, but no longer enough to carry all of SS13. It also has new prettified sprites and can now fit on belt slots.
- The soulstone item has been replaced with a malfunctioning necrostone- it can resurrect more active entities than the typical stone, but it grants no loyalties.
- The cowhand's gun, Slowpoke, can now be found in the necropolis's treasure caches! This weapon is self-charging, and the sparkling blasts it creates can pierce both rock and flesh. When its cell is low, the power it projects grows considerably.
- A hermit's shawl can now be found in the necropolis's loot- it provides an increase to movement speed, protection from temperature, and a rapidly-recharging magical barrier. However, be not overconfident: When your shield is down, the damage you take from attacks is **considerably** amplified.
- A curiously fashionable outfit has appeared **amoung** the loot of the necropolis...
- Ports the tribal torch from F13, can be crafted from wood in the tribal crafting tab. Use it in hand to plant it in the ground, then hit it with something hot to light it.

## Why It's Good For The Game
Adds some fun new loot to the pool.
Reworks some of the weaker or more unabashedly overpowered items in the necro pool, making the lives of aspiring treasure hunters easier.
Barely begins the long process of redesigning necroloot to both be more stable and more engaging.
Next is probably more nerfs/reworks(enjoy your ERT-level no slowdown armor while it lasts), and a buff to tendrils themselves.

## Changelog
:cl:
add: A particularly drippy outfit. Don't take it off!
add: The ascetic's garb, an overwear item that speeds up the wearer and surrounds them in a protective desert wind. Singificantly increases damage taken when shield is down.
add: Ports the Ice Cube artifact from /TG/
add: Ports spur from Yogstation, reworks it slightly to rebalance it to mining pool standards.
add: Ports tribal torch from F13 and adds the associated crafting recipe
add: New cursed katana sprites and aesthetic.
add: A number of artefact item icons ported from modern /TG/ 
add: Concussion Gauntlets! Special martial art content incoming.
tweak: Replaces soulstone with a nerfed necrostone
tweak: Assigns new sprites to the Paradox bag, and nerfs its storage capacity
tweak: A mining scanner has been added to the diamond pickaxe necroloot drop.
tweak: An envy's knife has been added to the voodoo doll drop. (thematic appropriateness)
balance: The cursed heart's "beat" ability can now be triggered when stunned or cuffed. Bloodloss from missing a beat has been halved. Pending possible nerfs to heal amount if this is too busted.
balance: Redesigned immortality talisman to be more generally useful, and to accept a custom BIG SCARY SHOUT
balance: Redesigns Hook Shot item to apply fast attacks in melee range. 
tweak: Added new pathing to some necro items, and reorganized list to contain two less items (for now)
balance: Removed Jacob's Ladder and Dark Blessing from the necropolis loot pool.
balance: Added buffed rework of possessed blade to necroloot pool, gaining amplified damage, block chance, and bloodletting
balance: Reworks cursed blade to cause acts of hubris and occasionally painful death
balance: Buffed all the elite items from useless to ok, since you can only get them in the first place by fighting a pvp megafauna
tweak: gave the necropolis chest power miner a small chance to drop a chooseable version, instead. Spin the wheel of overwhelmingly low chances.
tweak: removes ability to choose gravokinetic guardian until the game-breaking visual bugs are fixed.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -223,13 +223,51 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
-	slot_flags = null
-
 /obj/item/katana/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku!</span>")
 	return(BRUTELOSS)
 
+/obj/item/katana/cursed
+	name = "ominous katana"
+	desc = "An ancient single-edged weapon, possessing a flawless sheen despite it's clear age. The blade seems almost too perfect, and passes through most substances like water. <span class='warning'><b>Something is wrong.</b></span>."
+	force = 50
```
Comment (ZephyrTFA) [inline code/game/objects/items/weaponry.dm]: Far too much damage, reduce it and make the faction bonus larger

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -223,13 +223,51 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	armour_penetration = 30
+	max_integrity = 500
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	var/hunger = 0//Fill your blade to get a bonus(If I can safely implement this I will nerf the damage slightly, and boost the selfdam)
+	var/list/nemesis_factions = list("mining", "boss")
+	var/faction_bonus_force = 15//Even with the highest damage of any practically accessible weapon, it still can't reliably kill some fauna without putting people into crit. Just mining balance I guess
+
+/obj/item/immortality_talisman/examine(mob/user)
```
Comment (AsciiSquid) [inline code/game/objects/items/weaponry.dm]:
```suggestion
/obj/item/katana/cursed/examine(mob/user)
```
Assuming that's a typo

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (AsciiSquid) [reply code/game/objects/items/weaponry.dm]: I suggest `force = 20` and `faction_bonus_force = 30` to still meet that 50 damage bar while keeping it on-par with other swords.

**Review by AsciiSquid — CHANGES_REQUESTED** [review]

**Comment by Sun-Soaked** [comment]
This is ready for testmerging- I need help from more accomplished coders on the Katana mechanics and the Earthquake Stance, but both have a gutted fully-functional version in this code. Everything else should be good to go.

**Comment by Sun-Soaked** [comment]
time to see if merging up fixed the funkiness

**Comment by Sun-Soaked** [comment]
thank christ

File: code/datums/martial/earthquake.dm [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+	REMOVE_TRAIT(H, TRAIT_NOGUNS, EARTHQUAKE_TRAIT)
+	REMOVE_TRAIT(H, TRAIT_PIERCEIMMUNE, EARTHQUAKE_TRAIT)
+	REMOVE_TRAIT(H, TRAIT_STUNRESISTANCE, EARTHQUAKE_TRAIT)
+	REMOVE_TRAIT(H, TRAIT_IGNOREDAMAGESLOWDOWN, EARTHQUAKE_TRAIT)
+	H.physiology.brute_mod = initial(H.physiology.brute_mod)
+	H.physiology.burn_mod = initial(H.physiology.burn_mod)
+	H.physiology.stamina_mod = initial(H.physiology.stamina_mod)
+	H.physiology.stun_mod = initial(H.physiology.stun_mod)
+	H.physiology.pressure_mod = initial(H.physiology.pressure_mod)
+	H.physiology.cold_mod = initial(H.physiology.cold_mod)
```
Comment (ZephyrTFA) [inline code/datums/martial/earthquake.dm:138]: you forgot to also set heat_mod to intial

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

File: code/datums/martial/earthquake.dm (hunk 3 above)
Comment (Sun-Soaked) [reply code/datums/martial/earthquake.dm:138]: I'll fix it RQ, but note this is a WIP version and the proper implementation of EQS will likely be part of my port of newtg's martial art framework

File: code/datums/mood_events/generic_positive_events.dm [hunk 4]
```diff
@@ -175,6 +175,18 @@
...
+/datum/mood_event/drippy
+	description = "<span class='nicegreen'>My self confidence, unstopping. My style, unbreakable.</span>\n"
+	mood_change = 5
+
+/datum/mood_event/dripjordan
+	description = "<span class='nicegreen'>These shoes make me feel like I could really push myself to victory!</span>\n"
+	mood_change = 5
+
+/datum/mood_event/nojordans
+	description = "<span class='warning'>They're gone... my fasion is ruined. I can feel my self esteem decaying... </span>\n"
```
Comment (MarkSuckerberg) [inline code/datums/mood_events/generic_positive_events.dm]:
```suggestion
	description = "<span class='warning'>They're gone... my fashion is ruined. I can feel my self esteem decaying... </span>\n"
```

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 5]
```diff
@@ -746,3 +746,7 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 
 /obj/item/guardiancreator/miner/choose
 	random = FALSE
+	name = "glimmering shard"
+	desc = "Seems to be a very old rock, may have originated from a strange meteor. <b>This one looks exceptionally pure.</b>"
+	possible_guardians = list("Assassin", "Chaos", "Charger", "Dextrous", "Explosive", "Lightning", "Protector", "Ranged", "Standard", "Support	")
```
Comment (MarkSuckerberg) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]:
```suggestion
	possible_guardians = list("Assassin", "Chaos", "Charger", "Dextrous", "Explosive", "Lightning", "Protector", "Ranged", "Standard", "Support")
```

**Review by MarkSuckerberg — APPROVED** [review]
This is some good work. 👍👍👍

**Pull request merged**